src/parser.ts

```diff
@@ -164,82 +179,171 @@ class Parser {
+        })
+
+        return ({
+            fullComment: (mainComment + '\n' + tagComments.join('\n')).trim(),
+            tags: tagMap,
+            description: mainComment
         });
+    }
+
+    extractDefaultPropsFromComponent(symbol: ts.Symbol, source: ts.SourceFile) {
+        const possibleStatements = source.statements.filter(statement => this.checker.getSymbolAtLocation((statement as ts.ClassDeclaration).name) === symbol);
+        if (!possibleStatements.length) {
+            return {};
+        }
+        const statement = possibleStatements[0];
+        if (statementIsClassDeclaration(statement) && statement.members.length) {
+            const possibleDefaultProps = statement.members.filter(member => member.name && getPropertyName(member.name) === 'defaultProps');
+            if (!possibleDefaultProps.length) {
+                return {};
+            }
+            const defaultProps = possibleDefaultProps[0];
+            const { initializer } = (defaultProps as ts.PropertyDeclaration);
+            const { properties } = (initializer as ts.ObjectLiteralExpression);
+            const propMap = (properties as ts.NodeArray<ts.PropertyAssignment>).reduce((acc, property) => {
+                const literalValue = getLiteralValueFromPropertyAssignment(property);
+                if (typeof literalValue === 'string') {
+                    acc[getPropertyName(property.name)] = getLiteralValueFromPropertyAssignment(property);
+                }
+                return acc;
+            }, {} as StringIndexedObject<string>)
+            return propMap;
+        }
+        return {};
+    }
+}
+
+function statementIsClassDeclaration (statement: ts.Statement): statement is ts.ClassDeclaration {
+    return !!(statement as ts.ClassDeclaration).members;
+}
+
+function getPropertyName(name: ts.PropertyName): string | null {
+    switch (name.kind) {
+        case ts.SyntaxKind.NumericLiteral:
+        case ts.SyntaxKind.StringLiteral:
+        case ts.SyntaxKind.Identifier:
+            return name.text;
+        case ts.SyntaxKind.ComputedPropertyName:
+            return name.getText();
+        default:
+            return null;
+    }
+}
+
+function getLiteralValueFromPropertyAssignment(property: ts.PropertyAssignment): string | null {
+    const { initializer } = property;
+    switch(initializer.kind) {
+        case ts.SyntaxKind.FalseKeyword:
+            return 'false';
+        case ts.SyntaxKind.TrueKeyword:
+            return 'true';
+        case ts.SyntaxKind.StringLiteral:
+            return (initializer as ts.StringLiteral).text.trim();
+        case ts.SyntaxKind.NumericLiteral:
+            return `${(initializer as ts.NumericLiteral).text}`;
+        case ts.SyntaxKind.NullKeyword:
+            return 'null';
+        case ts.SyntaxKind.Identifier:
+            // can potentially find other identifiers in the source and map those in the future
```

Comment:
`Identifier` seems to be used for a variable or `undefined`. in the future, can probably grab literal values from the source doc but, while nice, seemed outside of the scope of this PR